cadc-sia/src/main/java/ca/nrc/cadc/sia2/impl/ServiceAvailability.java

```diff
@@ -149,23 +149,32 @@ public void setState(String string) {
         //no-op
     }
 
+    private static boolean isIVOURI(final URI uri) {
```

Comment:
there will never (> 99%) be a more elaborate check here so please inline this where it is used

Reply:
Done.